source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRRegistration.mm

```diff
@@ -118,16 +126,21 @@ - (void)setActionSetRenderer:(id<ACRIBaseActionSetRenderer>)actionsetRenderer
     _actionSetRenderer = actionsetRenderer;
 }
 
-- (void) setActionRenderer:(ACRBaseActionElementRenderer *)renderer cardElementType:(NSNumber *)cardElementType
+- (void)setActionRenderer:(ACRBaseActionElementRenderer *)renderer cardElementType:(NSNumber *)cardElementType
 {
+    // custom action must be registered through set custom action renderer method
+    if(cardElementType.longValue > ACRUnknownAction) {
```

Comment:
>cardElementType.longValue > ACRUnknownAction [](start = 7, length = 44)

If this is a requirement of the enum, then at least put a comment in the enum that the order is important and that the ACRUnknownAction needs to always be last.